Sources/Networking/HTTPClient/HTTPClient.swift

```diff
@@ -420,10 +420,6 @@ private extension HTTPClient {
             .asOptionalResult?
             .convertUnsuccessfulResponseToError()
 
-        self.trackHttpRequestPerformedIfNeeded(request: request,
-                                               requestStartTime: requestStartTime,
-                                               result: result)
-
```

Comment:
The code would not reach this point when receiving a network error (due to some early returns in the `parse` method --> https://github.com/RevenueCat/purchases-ios/blob/1477e8ca9768e12027bcf5dd519e4157f7dbcb5b/Sources/Networking/HTTPClient/HTTPClient.swift#L334-L343)

Reply:
Ohhh true, nice catch!